components/common/TableOrderDetails/index.tsx

```diff
@@ -1,53 +1,85 @@
+import { ethers } from 'ethers'
+import Image from 'next/image'
+
+import { useMarketplaceContext } from '@contexts/MarketplaceContext'
 import { Order } from '@typez/blockchain/Order'
 
+import EpicImg from '../../../public/images/other/epic.png'
+import GenesisImg from '../../../public/images/other/genesis.png'
+import HeroicImg from '../../../public/images/other/heroic.png'
+import LegendaryImg from '../../../public/images/other/legendary.png'
+import RareImg from '../../../public/images/other/rare.png'
 import { Styles, PriceText, EventText, FromText } from './styles'
 
+
 interface Props {
   orders: Order[]
 }
 
-const TableOrderDetails: React.FC<Props> = ({ orders }: Props) => (
-  <Styles>
-    <table>
-      <tr>
-        <th>ID</th>
-        <th className='image-th'>Image</th>
-        <th>Name</th>
-        <th>$MHT</th>
-        <th>$BUSD</th>
-      </tr>
-      {orders.map((order) => {
-        const { id } = order
-        const href = `/marketplace/orders/${order.id.toString()}`
-        return (
-          <tr key={id.toString()}>
-            <td className='id-td'>
-              <FromText>
-                <a href={href}>{id.toString()}</a>
-              </FromText>
-            </td>
-            <td className='image-td'>
-              <a href={href}>
-                {/* TODO: Adjust image (Pedro) <Image alt='alt' width='100' height='100' src={image} /> */}
-              </a>
-            </td>
-            <td className='name-td'>
-              <EventText>
-                <a href={href}>name</a>
-              </EventText>
-            </td>
-            <td className='mht-td'>
-              {/* TODO: Pedro vai seguir essa task */}
-              <PriceText>mht</PriceText>
-            </td>
-            <td className='usd-td'>
-              <PriceText>busd</PriceText>
-            </td>
-          </tr>
-        )
-      })}
-    </table>
-  </Styles>
-)
+const boostersImage = {
+  LEGENDARY: LegendaryImg,
+  RARE: RareImg,
+  HEROIC: HeroicImg,
+  GENESIS: GenesisImg,
+  EPIC: EpicImg
+}
+
+type Rarity = 'LEGENDARY' | 'RARE' | 'HEROIC' | 'GENESIS' | 'EPIC'
```

Comment:
Apenas colocar um comentário de TODO ou já importar algum tipo que já existe lá
```suggestion
//TODO change to import BoosterTypes or move this type to types folder
type Rarity = 'LEGENDARY' | 'RARE' | 'HEROIC' | 'GENESIS' | 'EPIC'
```